store/sqlstore/webhook_store.go

```diff
@@ -175,9 +175,29 @@ func (s SqlWebhookStore) PermanentDeleteIncomingByChannel(channelId string) *mod
 }
 
 func (s SqlWebhookStore) GetIncomingList(offset, limit int) ([]*model.IncomingWebhook, *model.AppError) {
+	return s.GetIncomingListByUser("", offset, limit)
+}
+
+func (s SqlWebhookStore) GetIncomingListByUser(userId string, offset, limit int) ([]*model.IncomingWebhook, *model.AppError) {
 	var webhooks []*model.IncomingWebhook
 
-	if _, err := s.GetReplica().Select(&webhooks, "SELECT * FROM IncomingWebhooks WHERE DeleteAt = 0 LIMIT :Limit OFFSET :Offset", map[string]interface{}{"Limit": limit, "Offset": offset}); err != nil {
+	query := s.getQueryBuilder().
+		Select("*").
+		From("IncomingWebhooks").
+		Where(sq.And{
```

Comment:
nit: if there's just one condition, I think you can eliminate the `sq.And`

Reply:
I wasn't sure if I still need it so it knew what operator to use if it added the additional `userId` condition below. I'm guessing it will default to `AND` if nothing is provided? Looking into it

Reply:
@lieut-data I've made that last change. If you are good with the latest commit I think it would be ready to merge.